docs/getting-started/quickstart.md

```diff
@@ -34,32 +41,36 @@ At this point you can test if your build has worked correctly by typing **make t
 * **redis-sentinel** is the Redis Sentinel executable (monitoring and failover).
 * **redis-cli** is the command line interface utility to talk with Redis.
 * **redis-benchmark** is used to check Redis performances.
-* **redis-check-aof** and **redis-check-rdb** (**redis-check-dump** in 3.0 and below) are useful in the rare event of corrupted data files.
+* **redis-check-aof** and **redis-check-rdb** (**redis-check-dump** in version 3.0 and below) are useful in the rare event of corrupted data files.
 
 It is a good idea to copy both the Redis server and the command line interface into the proper places, either manually using the following commands:
 
-* sudo cp src/redis-server /usr/local/bin/
-* sudo cp src/redis-cli /usr/local/bin/
+    sudo cp src/redis-server /usr/local/bin/
+    sudo cp src/redis-cli /usr/local/bin/
 
 Or just using `sudo make install`.
 
-In the following documentation we assume that /usr/local/bin is in your PATH environment variable so that you can execute both the binaries without specifying the full path.
+In the following documentation we assume that `/usr/local/bin` is in your `PATH` environment variable so that you can execute both the binaries without specifying the full path.
```

Comment:
```suggestion
In the following documentation we assume that `/usr/local/bin` is in your `PATH` environment variable so that you can execute both of the binaries without specifying the full path.
```